src/utils/sortPeople.ts

```diff
@@ -0,0 +1,46 @@
+import { Person } from '../types';
+import { SortBy, SortOrder } from '../types/Sort';
+
+export function sortPeople(
+  people: Person[],
+  field: keyof Person,
+  order: string,
+): Person[] {
+  const sorted = [...people];
+
+  switch (order) {
+    case SortOrder.asc:
+      if (field === SortBy.name || field === SortBy.sex) {
+        sorted.sort((a, b) => {
+          return a[field].localeCompare(b[field]);
+        });
+      } else {
+        sorted.sort((a, b) => {
+          const n1 = a[field] ? +a[field] : 0;
+          const n2 = b[field] ? +b[field] : 0;
+
+          return n1 - n2;
+        });
+      }
```

Comment:
The sorting logic inside the `switch` statement is duplicated for both `SortOrder.asc` and `SortOrder.decs` cases. Consider abstracting this logic into a separate function to avoid code duplication. This function could take the sorting order as a parameter and return the appropriate sorting function.